Add provisioning job to nightly kyma on aks (#642)
kyma-project/test-infra: docs/prow/presets.md

| **preset-gc-compute-envs**         | It provides environment variables with the GCP compute zone and the GCP compute region.                                                                   |
| **preset-sa-vm-kyma-integration**  | It injects credentials for the service account to run integration tests on virtual machines (VMs).                                                              |
| **preset-sa-gke-kyma-integration** | It injects credentials for the service account to run integration tests on a Google Cloud Engine (GKE) cluster.                                                 |
| **preset-creds-aks-kyma-integration** | It injects credentials to run integration tests on an Azure Kubernetes Service (AKS) cluster. |

> | **preset-nightly-aks-github-integration** | It injects github application credentials to enable github authentication via dex for AKS cluster |

Reviewer: ```suggestion
| **preset-nightly-aks-github-integration** | It injects GitHub application credentials which enable GitHub authentication for an AKS cluster using Dex. |
```
